lib/api/bucketGet.js

```diff
@@ -241,9 +301,17 @@ function bucketGet(authInfo, request, log, callback) {
         listingType: 'DelimiterMaster',
         maxKeys: actualMaxKeys,
         delimiter: params.delimiter,
-        marker: params.marker,
         prefix: params.prefix,
     };
+    if (v2) {
```

Comment:
Based on the docs, perhaps the condition should be that `v2` equals `2` rather than any truthy value.

Reply:
I do follow that, I mean that this `if` block condition could check `list-type` is set to `2` rather than any truth value of `list-type`. That way, for example, a user couldn't set `list-type` to `1` and then get the v2 listing.

The next line which sets `listParams.v2 = true;` is all good for me 😄.

Reply:
ooh gotcha, yeah that's actually a good point

Reply:
I updated this by including a check above that v2 is actually 2

Reply:
Nice, LGTM.